package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "oppia",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "dependencies": {
+    "@mapbox/stylelint-processor-arbitrary-tags": "^0.2.0",
```

Comment:
Pin to specific version, not a bunch of them

Reply:
Done.